### ISSUE TITLE
[BUG][protobuf-schema] uniqueItems created invalid import statement

### ISSUE THREAD

**Issue by JulianGmp** [OpenAPITools/openapi-generator#11482]
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x] Have you searched for related issues/PRs?
- [x] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description

When enabling the `uniqueItems` flag on an array in a schema, the generated proto files will include a non existend `set.proto` file.

##### openapi-generator version

output from `--version`
```
openapi-generator-cli 6.0.0-SNAPSHOT
  commit : 8a2131f
  built  : 2022-02-01T15:07:00+01:00
  source : https://github.com/openapitools/openapi-generator
  docs   : https://openapi-generator.tech/
```

##### OpenAPI declaration file content or url

```yaml
openapi: 3.0.0
info:
  title: Sample API
  description: Optional multiline or single-line description in [CommonMark](http://commonmark.org/help/) or HTML.
  version: 0.1.9

servers:
  - url: http://api.example.com/v1
    description: Optional server description, e.g. Main (production) server

components:
  schemas:
    userinfo:
      properties:
        idontknow:
          type: array
          uniqueItems: true
          items:
            type: integer


paths:
  /users:
    get:
      summary: Returns a list of users.
      description: Optional extended description in CommonMark or HTML.
      responses:
        '200':
          description: A JSON array of user names
          content:
            application/json:
              schema:
                $ref: '#components/schemas/userinfo'
```

##### Generation Details

Generated `userinfo.proto`:
```proto
/*
  Sample API

  Optional multiline or single-line description in [CommonMark](http://commonmark.org/help/) or HTML.

  The version of the OpenAPI document: 0.1.9

  Generated by OpenAPI Generator: https://openapi-generator.tech
*/

syntax = "proto3";

package openapitools;

import public "models/set.proto"; // << this breaks

message Userinfo {

  repeated int32 idontknow = 371509509;

}
```

##### Steps to reproduce

Generate the protobuf-schema with the given yaml input.

##### Suggest a fix

I would expect the unique items flag to be ignored, or maybe mentioned in a comment, in the generated proto files, since protobuf is a pure container format, and it provides no way for us to validate the unique rule.

**Comment by Soneca-ii** [comment]
Is there a workaround for this bug?

**Comment by wing328** [comment]
Should be fixed by #13286. Please pull the latest master to give it a try.
